test/consensus-gateway/hash_kernel_intent.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+
+const ConsensusGatewayBase = artifacts.require('ConsensusGatewayBaseTest');
+
+contract('ConsensusGatewayBase::hashKernelIntent', (accounts) => {
+  it('should hash kernel content', async () => {
+    const consensusGatewayBase = await ConsensusGatewayBase.new();
+
+    const height = new BN(1);
```

Comment:
```suggestion
    const height = new BN(Utils.getRandomNumber(500));
```

Reply:
Created discussion point can be discussed with the team. 
https://hackmd.io/1rni8bzWQo2_wnqrpDn59Q